frame/example-offchain-worker/src/lib.rs

```diff
@@ -334,7 +335,7 @@ impl<T: Trait> Module<T> {
 
 		// Make an external HTTP request to fetch the current price.
 		// Note this call will block until response is received.
-		let price = Self::fetch_price().map_err(|e| format!("{:?}", e))?;
+		let price = Self::fetch_price().map_err(|_| "Failed to fetch price")?;
```

Comment:
I wonder if `lite_format` also exists

Reply:
Probably something will exists. However we should really not encourage people to use `String`s in the runtime^^